src/OpenTelemetry.Exporter.Prometheus/PrometheusExporterExtensions.cs

```diff
@@ -47,7 +47,7 @@ public static void WriteMetricsCollection(this PrometheusExporter exporter, Stre
             foreach (var metric in exporter.Metrics)
             {
                 var builder = new PrometheusMetricBuilder()
-                    .WithName(metric.Name)
+                    .WithName(metric.Meter.Name + metric.Name)
```

Comment:
Prometheus doesn't have concept of Meter (like OTLP does).. Trying to see if this is a good approach to use the meter name as namespace, to avoid name collisions, when same instrument name is used across multiple instruments, from different Meter.